controllers/awardsController.js

```diff
@@ -0,0 +1,6 @@
+export default `'SELECT *
+From awards a
+INNER JOIN awards_linking al
+    USING(award_id)
+INNER JOIN films f1
+    USING(film_id)`;
```

Comment:
Change this USING(film_id) to a ON statement:
`ON al.film_id = f1.film_id`

Needed since the awards_linking has the film_id not the awards table